src/modals/Eject.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import WarningAmberIcon from "@mui/icons-material/WarningAmber";
+import {
+  Box,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  Typography,
+} from "@mui/material";
+
+interface EjectProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const EjectModal: React.FC<EjectProps> = ({ isOpen, onClose }) => {
+  return (
+    <Dialog
+      open={isOpen}
+      onClose={onClose}
+      aria-labelledby="alert-dialog-title"
+      aria-describedby="alert-dialog-description"
+    >
+      <DialogTitle id="alert-dialog-title">
+        <WarningAmberIcon
+          style={{ marginRight: "10px", verticalAlign: "middle" }}
+        />
+        Eject Intermediary
+      </DialogTitle>
+      <DialogContent>
+        <Box>
+          <Typography variant="h6" color="error">
+            Eject feature is not fully implemented
+          </Typography>
+          <br></br>
+          <Typography variant="body1">
+            If implemented, this would initiate a challenge where the
+          </Typography>
+          <Typography variant="body1">
+            SCBridge-Wallet owner sends its latest known state on-chain
+          </Typography>
+          <Typography variant="body1">
+            and the Intermediary has a timeout period where they can{" "}
+          </Typography>
+          <Typography variant="body1">
+            respond with their latest state if it has a higher turn number.
+          </Typography>
```

Comment:
What happens to this text if it's all in one Typography element?

Reply:
I thought it would force all the text to show on the same line but looks like that's not the case. Will use a single `Typography` element